.env

```diff
@@ -0,0 +1,3 @@
+PORT=3006
```

Comment:
Avoid pushing your .env files into your repos so that all your sensitive variables aren't vulnerable

Reply:
my .env file doesn't have anything valuable in it. does it matter if i include or not include it then?

Reply:
If we were to look at it from the context of a 'scalable' app, at some point in time if you did want to expand this service and later on might probably add environmental variables then it would be a good idea to ignore it in the beginning, ultimately up to you though. It's just good practice to have